Lit_swift/Core/LitClient-executeJs.swift

```diff
@@ -12,9 +12,9 @@ extension LitClient {
     /**
      * Execute JS on the nodes and combine and return any resulting signatures
      */
-    func executeJs(code: String?, ipfsId: String?, authSig: [String: Any]?,  sessionSigs: [String: Any]?, authMethods: [AuthMethod]?, jsParams: [String: Any], debug: Bool = true) -> Promise<Any> {
+    func executeJs(code: String?, ipfsId: String?, authSig: [String: Any]?,  sessionSigs: [String: Any]?, authMethods: [AuthMethod]?, jsParams: [String: Any], debug: Bool = true) -> Promise<[String: Any]> {
         guard self.isReady else {
-            return Promise(error: LitError.litNotReady)
+            return Promise(error: LitError.litNodeClientNotReady)
```

Comment:
Should be `LitNodeClientNotReadyError`: https://github.com/LIT-Protocol/js-sdk/blob/master/packages/constants/src/lib/errors.ts#L27

Reply:
Please update all the instances of `LitNodeClientNotReady` to the above